Sources/Foundation/FileManager+Win32.swift

```diff
@@ -412,6 +412,28 @@ extension FileManager {
         }
         return substitutePath
     }
+    
+    private func _nonThrowingDestinationOfSymbolicLink(atPath path: String) -> String {
```

Comment:
I think that this is poorly named, as it is returning `path` rather than an error.  Perhaps it makes sense to call this `realpath`?

Reply:
Done.